### PR TITLE
user address addition and adding full address to seed data

### PR DESCRIPTION
**Pull request by alejandro-loja**
user address addition and adding full address to seed data

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -68,117 +86,938 @@ const setUp = async () => {
...
-    Product.create({name: "Goya Mango Nectar", description: faker.random.words(20), price: 2.99, category: 'mexico', imgUrl: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS1-lkmlXqFICtX4Xql9uTJN7_kjYvI19MVBC0uqEzfIOtq0FRFtSd2dKpP1BAHyBFKJWc&usqp=CAU", ml: 250}),
-    Product.create({name: "Goya Coconut Milk", description: faker.random.words(20), price: 2.99, category: 'mexico', imgUrl: "https://images.heb.com/is/image/HEBGrocery/001964546", ml: 250}),
-    Product.create({name: "Goya Aloe Vera Original", description: faker.random.words(20), price: 2.99, category: 'mexico', imgUrl: "https://images.epallet.com/production/media/cache/5f/90/5f9022d584bd248d1d1d599c557bb980.jpg", ml: 250}),
-    Product.create({name: "Goya Coconut Water", description: faker.random.words(20), price: 2.99, category: 'mexico', imgUrl: "https://cdn.shopify.com/s/files/1/0280/6834/8007/products/GY2778_1024x1024.jpg?v=1645526560", ml: 250}),
-    Product.create({name: "Malta Goya", description: faker.random.words(20), price: 2.99, category: 'mexico', imgUrl: "https://i5.peapod.com/c/E9/E9DUK.jpg", ml: 250}),
-    ])
+      //'philippines'
```
Comment (alejandro-loja) [inline server.js:89]: from here on to the bottom it is just "prettier" automatic formating.

**Review by Banychen — APPROVED** [review]
Looks Good!

File: src/store/cart.js [hunk 2]
```diff
@@ -9,40 +9,48 @@ const cart = (state = { lineItems: [] }, action) => {
...
-//     item = (
-//       await axios.put(`/api/orders/cart`, item, {
-//         headers: {
-//           authorization: window.localStorage.getItem("token"),
-//         },
-//       })
-//     ).data;
-//     dispatch({ type: "UPDATE_CART_ITEM", item });
-//   };
-// };
```
Comment (bob-zs) [inline src/store/cart.js:45]: Delete the comment code please

**Review by bob-zs — APPROVED** [review]

File: src/store/cart.js (hunk 2 above)
Comment (alejandro-loja) [reply src/store/cart.js:45]: deleted , good call

**Pull request merged**